automated-comments/elixir/solution/use_module_doc.md

```diff
@@ -0,0 +1,8 @@
+<!--
+This error is raised when the submission lacks @moduledoc for the Module
+-->
+
+It is recommended to have a [`@moduledoc`](https://hexdocs.pm/elixir/writing-documentation.html#module-attributes) attribute to describe the intent of the Module.
```

Comment:
```suggestion
It is recommended to have a [`@moduledoc`](https://hexdocs.pm/elixir/writing-documentation.html#module-attributes) attribute to describe the intent of the `Module`.
```

Reply:
`Module` is a certain module of the elixir standard library, so printing it as code might be confusing.

I'd suggest to just lowercase it:

```suggestion
It is recommended to have a [`@moduledoc`](https://hexdocs.pm/elixir/writing-documentation.html#module-attributes) attribute to describe the intent of the module.
```